future-proofing.yaml

```diff
@@ -23,31 +23,50 @@ future-proofing:
+        - bin/future add-set \Rector\Set\ValueObject\SetList::PRIVATIZATION
+        - bin/future add-set \Rector\Set\ValueObject\SetList::TYPE_DECLARATION
+        - bin/future add-set \Rector\Set\ValueObject\SetList::TYPE_DECLARATION_STRICT
+        - bin/future add-set \Rector\Set\ValueObject\LevelSetList::UP_TO_PHP_82
+
+        # section that registers rector rules
+        - bin/future add-rule \Rector\DependencyInjection\Rector\Class_\ActionInjectionToConstructorInjectionRector
+        - bin/future add-rule \Rector\RemovingStatic\Rector\ClassMethod\LocallyCalledStaticMethodToNonStaticRector
+        - bin/future add-rule \Rector\Restoration\Rector\Property\MakeTypedPropertyNullableIfCheckedRector
+        - bin/future add-rule \Rector\Restoration\Rector\Class_\RemoveFinalFromEntityRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ArrowFunction\AddArrowFunctionReturnTypeRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\Closure\AddClosureReturnTypeRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\AddMethodCallBasedStrictParamTypeRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\AddReturnTypeDeclarationBasedOnParentClassMethodRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\AddVoidReturnTypeWhereNoReturnRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\ParamTypeByMethodCallTypeRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\ParamTypeByParentCallTypeRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\Param\ParamTypeFromStrictTypedPropertyRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromReturnNewRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictBoolReturnExprRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictNativeCallRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictNewArrayRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictTypedPropertyRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\Property\TypedPropertyFromAssignsRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\Property\TypedPropertyFromStrictConstructorRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\Property\TypedPropertyFromStrictGetterMethodReturnTypeRector
+        - bin/future add-rule \Rector\TypeDeclaration\Rector\Property\TypedPropertyFromStrictSetUpRector
+        - bin/future add-rule \Rector\Visibility\Rector\ClassMethod\ExplicitPublicClassMethodRector
+
+        #section that excludes rector rules
+        - bin/future skip \Rector\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector
```

Comment:
@calinpristavu same question

This is where it becomes mandatory because we need to make it like this:
![image](https://user-images.githubusercontent.com/20539991/203615178-7eeca085-7d51-42e5-b428-e3b1ea6dcae9.png)

and not like this (this won't work)
![image](https://user-images.githubusercontent.com/20539991/203615422-56416447-2843-42a7-9637-aa85e93cb4b6.png)

Reply:
Same answer ^_^